packages/hash/external-services/kratos/kratos.yml

```diff
@@ -0,0 +1,61 @@
+# Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
+
+version: v0.7.1-alpha.1
+
+dsn: memory
+
+serve:
+  public:
+    base_url: http://localhost:4433/
+    cors:
+      enabled: true
+  admin:
+    base_url: http://kratos:4434/
+
+selfservice:
+  default_browser_return_url: http://localhost:3000/
+  allowed_return_urls:
+    - http://localhost:3000
+
+  methods:
+    password:
+      enabled: true
+
+  flows:
+    error:
+      ui_url: http://localhost:3000/error
+
+    logout:
+      after:
+        default_browser_return_url: http://localhost:3000/login
+
+    login:
+      ui_url: http://localhost:3000/login
+      lifespan: 10m
+
+    registration:
+      lifespan: 10m
+      ui_url: http://localhost:3000/signup
+      after:
+        password:
+          hooks:
+            - hook: session
+
+log:
+  level: debug
+  format: text
+  leak_sensitive_values: true
+
+ciphers:
+  algorithm: xchacha20-poly1305
+
+hashers:
+  algorithm: bcrypt
```

Comment:
Just want to flag that we maybe should spend a little time thinking about the hasher. I think `argon2` is deemed "more secure", but unsure what that entails or by whom (just remember reading about it in passing)

Reply:
I do not have thoughts on this, and from skimming [this article](https://www.ory.sh/choose-recommended-argon2-parameters-password-hashing/) it appears setting this up may require some thought. I've added a todo in https://github.com/hashintel/hash/pull/889/commits/81a81a15779e8df5068a5727c418c30f288d8502, and created [a follow up task](https://app.asana.com/0/0/1202721575417478/f) to add support for this